zetaclient/maintenance/shutdown_listener.go

```diff
@@ -22,17 +26,32 @@ type ShutdownListener struct {
 	logger zerolog.Logger
 
 	lastRestartHeightMissed int64
+	getVersion              func() string
```

Comment:
Let's add some context indicating that this is zetacore's version

```suggestion
  // zetacore version resolver
	getNodeVersion              func() string
```

Reply:
This is not getting the node version, it's getting the zetaclient version